test/groovy/AbstractPiperUnitTest.groovy

```diff
@@ -0,0 +1,31 @@
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import com.sap.piper.DefaultValueCache
+import org.junit.Before
+import org.yaml.snakeyaml.Yaml
+
+import static com.lesfurets.jenkins.unit.global.lib.LibraryConfiguration.library
+
+class AbstractPiperUnitTest extends BasePipelineTest {
```

Comment:
There is already a base class merged in the meantime: test/groovy/PiperTestBase

Reply:
I changed it to use PiperTestBase